tests/test_basic_ops_list.py

```diff
@@ -738,6 +739,24 @@ def test_getattr() -> None:
     assert getattr(cfg, "2") == src[2]
 
 
+@mark.parametrize(
```

Comment:
`tests/test_basic_ops_list.py` seems like the wrong file for these tests.
move to test_basic_ops_dict ?

Reply:
thanks, yes, accidentally added to the wrong file :)